src/components/List/Section.js

```diff
@@ -24,28 +24,30 @@ const TitleContainer = styled(Row)`
 `;
 
 const TitleText = styled(BoldText)`
-  font-size: ${typography.size.uno};
-  color: ${colors.onyx.light};
-  text-transform: uppercase;
+  font-size: ${({ fontSize }) => fontSize || typography.size.uno};
+  color: ${({ color }) => color || colors.onyx.light};
+  text-transform: ${({ textTransform }) => textTransform || "uppercase"};
 `;
 
-const Section = ({ title, totalSections, children }) => (
-  <SectionContainer large={constants.MAX_COLUMNS / totalSections}>
+const Section = ({ title, noOfColumns, titleStyle, children, ...rest }) => (
+  <SectionContainer large={noOfColumns} {...rest}>
     <TitleContainer>
-      <TitleText>{title}</TitleText>
+      <TitleText {...titleStyle}>{title}</TitleText>
```

Comment:
Maybe pass `titleStyle` to `TitleText` as a `style` prop, rather than spreading its contained props? Then you could get rid of the destructured props within the `TitleText` `styled` block and leave the default values to be overwritten.